releasenotes/notes/submodule-update-a2f6c5e9cb597a99.yaml

```diff
@@ -0,0 +1,33 @@
+---
+critical:
+  - |
+    From this release on, a rcbops fork is used for the openstack-ansible
+    submodule. Due to this change, a pre-existing checkout of rpc-openstack will
+    require a submodule sync to ensure that the subodule is pointing to the
+    right repository. Failing to keep the submodule in sync will result in some
+    unexpected behavior, primarily you will be missing any code updates that
+    have been pushed up to the forked repository. To resolve this execute the
+    following:
+
+    .. code-block: shell
+      $ git submodule status
+      $ git submodule update --init
+      $ git submodule sync
+      $ git submodule status
+
+    The following is an example including output of an update from a
+    pre-existing tag to the HEAD of the rcbops/openstack-ansible repository:
+
+    .. code-block: shell
+      $ git submodule status
+      +2f4e89fcc1eba8437660bd1fcc3c0f45095d8f74 openstack-ansible (14.2.16)
+      $ git submodule update --init
+      Submodule path 'openstack-ansible': checked out '5644b36dbd74352d7503f809760a0684f0a84a9d'
+      $ git submodule sync
+      Synchronizing submodule url for 'openstack-ansible'
+      $ git submodule status
+      5644b36dbd74352d7503f809760a0684f0a84a9d openstack-ansible (newton-eol-1-g5644b36d)
```

Comment:
nit: I think it's clear enough to show the commands ane xample output, rather than repeat both.

Reply:
Agree, done